.env.example

```diff
@@ -3,6 +3,7 @@ LOCAL_SEED_ADDRESS=0xYOUR_DEV_ADDRESS
 HARDHAT_OWNER_ADDRESS=0xYOUR_DEV_ADDRESS
 REACT_APP_INFURA_PROJECT_ID=infuraidhere
 ETHEREUM_RPC_URL=https://mainnet.infura.io/v3/infuraidhere
+COORDINAPE_USER_ADDRESS=0xfad763da9051953fea58c2304395719a3b6ba361
```

Comment:
Is this address for staging or prod or local?

Reply:
this is for prod, can be the same address for staging as well